src/Sylius/Behat/Client/ResponseChecker.php

```diff
@@ -132,18 +132,54 @@ public function hasItemWithValue(Response $response, string $key, $value): bool
         return false;
     }
 
-    /** @param string|int $value */
-    public function hasSubResourceWithValue(Response $response, string $subResource, string $key, $value): bool
-    {
+    public function hasValueInAnySubresourceObjectCollection(
+        Response $response,
+        string $subResource,
+        string $key,
+        int|string $expectedValue,
+    ): bool {
```

Comment:
```suggestion
    public function hasSubresourceWithValuesInCollection(
        Response $response,
        string $subResource,
        /** @var array<string, string|int> $expectedValues */
        array $expectedValues,
    ): bool {
```
Would be nice to be able to pass an array of parameters that are expected, so we can check whether a singular object in the collection has a specific id/key/position/whatever at once.

Reply:
I've implemented the idea, please take a look if it's what you had in mind